### PR TITLE
IDEM-172

### PR DESCRIPTION
**Pull request by bitcoinbrisbane**
Run unit test on push

### CODE REVIEW

File: .github/workflows/ci.yml
Comment (xcjlo985) [inline .github/workflows/ci.yml:9]: Presume you want to revert this before merging

File: .github/workflows/ci.yml
Comment (xcjlo985) [inline .github/workflows/ci.yml:13]: I also presume if this runs fine we should update our standard to 20

File: package.json
Comment (xcjlo985) [inline package.json:35]: Dev dependency?

**Review by xcjlo985 — COMMENTED** [review]
I'll approve once the static checks pass. I checked this out and ran the unit tests and they are failing on my machine

File: .github/workflows/ci.yml
Comment (bitcoinbrisbane) [reply .github/workflows/ci.yml:13]: Yep, agreed.

File: package.json
Comment (bitcoinbrisbane) [reply package.json:35]: Fixed.

**Review by xiangzhe0812 — APPROVED** [review]

**Review by xcjlo985 — APPROVED** [review]

**Pull request merged**
